# 图像中所有检测到的姿态
std_msgs/Header header
yolo_detect/PoseEstimate[] poses